x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/servicenow_params.test.tsx

```diff
@@ -35,7 +35,8 @@ const editAction = jest.fn();
 const defaultProps = {
   actionConnector: connector,
   actionParams,
-  errors: { short_description: [] },
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  errors: { 'subActionParams.incident.short_description': [] },
```

Comment:
if you write it like this, there is no typescript error:
```
errors: { ['subActionParams.incident.short_description']: [] },
```

Reply:
Nice. 👍